tox.ini

```diff
@@ -54,7 +54,7 @@ deps =
 
     # Oldest deps we pin against.
     oldestdeps: sunpy<2.1
-    oldestdeps: scikit-image<0.17.0
+    oldestdeps: scikit-image<=0.18.0
```

Comment:
```suggestion
    oldestdeps: scikit-image<=0.19.0
```

Reply:
1.19 isn't out yet though right?

Reply:
Sure.

Reply:
I'm a little confused as to how `oldestdeps` works though. From what I understand, we test for every version up until < 0.19.0?

Reply:
It only tests one version.

Reply:
> It only tests one version.

Which version does it test in the above example?

Reply:
0.18.0